### PR TITLE
Switch masters to oopta

### PR DESCRIPTION
**Pull request by derekhiggins**

### CODE REVIEW

**Comment by derekhiggins** [comment]
Tested with the kni-rebase 
https://github.com/openshift-metalkube/kni-installer/pull/36
masters boot, ssh starts then ... nothing

**Comment by derekhiggins** [comment]
Build FAILURE, see build http://10.8.144.11:8080/job/dev-tools/522/

**Comment by derekhiggins** [comment]
Build FAILURE, see build http://10.8.144.11:8080/job/dev-tools/523/

File: 04_setup_ironic.sh [hunk 1]
```diff
@@ -28,36 +28,20 @@ done
...
-    sudo mount /dev/mapper/${LOOPBACK}p1 /tmp/mnt
-    sudo sed --follow-symlinks -i -e 's/ip=eth0:dhcp/ip=eth0:dhcp ip=eth1:dhcp/g' /tmp/mnt/grub2/grub.cfg
-    sudo umount /tmp/mnt
-    sudo kpartx -d /dev/${LOOPBACK}
-    sudo losetup -d /dev/${LOOPBACK}
-    qemu-img convert -O qcow2 -c "$RHCOS_IMAGE_FILENAME_RAW" "$RHCOS_IMAGE_FILENAME_DUALDHCP"
-    rm "$RHCOS_IMAGE_FILENAME_RAW"
+RHCOS_IMAGE_FILENAME_COMPRESSED=$RHCOS_IMAGE_FILENAME_BM_BIOS.qcow2
+if [ ! -e "$RHCOS_IMAGE_FILENAME_COMPRESSED" ] ; then
+    qemu-img convert -O qcow2 -c "$RHCOS_IMAGE_FILENAME_BM_BIOS" "$RHCOS_IMAGE_FILENAME_COMPRESSED"
```
Comment (vrutkovs) [inline 04_setup_ironic.sh]: Not sure why would `qemu-img convert` would be required here - isn't $RHCOS_IMAGE_FILENAME_BM_BIOS.qcow2 already decompressed?

**Comment by vrutkovs** [comment]
Interesting, bootstrap doesn't start temporary API server so `certs` pod on masters fails

**Comment by e-minguez** [comment]
> Interesting, bootstrap doesn't start temporary API server so `certs` pod on masters fails

To me it fails with:

```
Apr 04 08:25:19 kvirt-sprwork1.cloud.lab.eng.bos.redhat.com systemd[1]: Started Bootstrap a Kubernetes cluster.
Apr 04 08:25:38 kvirt-sprwork1.cloud.lab.eng.bos.redhat.com bootkube.sh[13949]: Rendering MCO manifests...
Apr 04 08:25:38 kvirt-sprwork1.cloud.lab.eng.bos.redhat.com bootkube.sh[13949]: I0404 08:25:38.690414       1 bootstrap.go:84] Version: 4.0.0-alpha.0-154-gc1ba218c-dirty
Apr 04 08:25:38 kvirt-sprwork1.cloud.lab.eng.bos.redhat.com bootkube.sh[13949]: F0404 08:25:38.691153       1 bootstrap.go:104] error rendering bootstrap manifests: open /assets/tls/etcd-metric-ca-bundle.crt: no such file or directory
Apr 04 08:25:38 kvirt-sprwork1.cloud.lab.eng.bos.redhat.com systemd[1]: bootkube.service: main process exited, code=exited, status=255/n/a
Apr 04 08:25:38 kvirt-sprwork1.cloud.lab.eng.bos.redhat.com systemd[1]: Unit bootkube.service entered failed state.
Apr 04 08:25:38 kvirt-sprwork1.cloud.lab.eng.bos.redhat.com systemd[1]: bootkube.service failed.
Apr 04 08:25:44 kvirt-sprwork1.cloud.lab.eng.bos.redhat.com systemd[1]: bootkube.service holdoff time over, scheduling restart.
Apr 04 08:25:44 kvirt-sprwork1.cloud.lab.eng.bos.redhat.com systemd[1]: Stopped Bootstrap a Kubernetes cluster.
```
The file is named `etcd-metrics-ca-bundle.crt` (notice the 's')

**Comment by vrutkovs** [comment]
@e-minguez works here w/ kni rebased PR

**Comment by vrutkovs** [comment]
Had to change bootstrap's /etc/resolv.conf from:
```
nameserver 127.0.0.1
nameserver 192.168.111.1
```
to
```
nameserver 192.168.111.2
```
to make `host -t SRV "_etcd-server-ssl._tcp.ostest.test.metalkube.org"` return valid SRV record

File: 04_setup_ironic.sh (hunk 1 above)
Comment (derekhiggins) [reply 04_setup_ironic.sh]: this is done to make the qcow2 file smaller, so it fits into /tmp on the masters (when Ironic is deploying it)

**Comment by derekhiggins** [comment]
Merging this as we have rebased kni-installer, and we should keep the RHCOS versions in sync with each other. We still have problems with bringing up the cluster but that can now be tackled.

**Pull request merged**